kubernetes/gke-base/docs/index.rst

```diff
@@ -206,12 +206,7 @@ Example output might look like::
 
 Verify Confluent Platform Control Center
```

Comment:
Include a screenshot of C3 showing the Connect page, you should see the Replicator connector running